src/EntryPoints/HookHandlers.php

```
@@ -17,18 +28,52 @@ public static function onExtensionRegistration(): void {
+
 	}
 
+	public static function onPageSaveComplete( WikiPage $wikiPage, MediaWiki\User\UserIdentity $user, string $summary, int $flags, MediaWiki\Revision\RevisionRecord $revisionRecord, MediaWiki\Storage\EditResult $editResult ) {
+        wfDebug( __METHOD__ . "SWB: onPageSaveComplete" );
+		// Access the semantic data of the page via the Semantic MediaWiki API
+        $semanticData = \SMW\MediaWiki\Hooks\ParserHooks::getSemanticDataForPage($wikiPage);
+
+        onSmwUpdateDataBefore(null, $semanticData);
+    }
+
+	public static function onMultiContentSave( RenderedRevision $renderedRevision, UserIdentity $user, CommentStoreComment $summary, $flags, Status $hookStatus ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)